genestack_client/genestack_shell.py

```diff
@@ -314,21 +318,22 @@ def process_command(self, command, argument_list, shell=False):
         try:
             args = p.parse_args(argument_list)
         except SystemExit:
-            return
+            return 1
         command.set_connection(self.connection)
         command.set_arguments(args)
         try:
-            return command.run()
+            command.run()
```

Comment:
why do we disable the possibility to return user-defined exit codes? And if the only option for the command to fail is throwing `GenestackException`, then we should at least mention it in the docstring of the `run` method